docusaurus/docs/themes/transportation/index.mdx

```diff
@@ -2,45 +2,129 @@ import DocCardList from '@theme/DocCardList';
-- The <code>during</code> property, if present, narrows the scope of the parent value to apply only during the time periods specified by the during value.
-- There is no negated version of the property, i.e. <code>notDuring</code>, because the phrase "not during X" can be expressed as "during the complement of X" and the OSM opening hours specification can handle all such cases.
+</p>
 
-### Environmental Scoping
+<p>
 
-There is a single environmental conditions scoping property named conditions which, if present, must contain a non-empty set of environmental condition values, modeled as an array containing unique items. These conditions include weather-related conditions (e.g. snow or fog), or local signaled conditions (e.g. OSM's flashing_light tag) that indicate some variation on the default properties of the transportation feature.
-The conditions property, if present, narrows the scope of the parent value to apply only when the local environment has at least one of the conditions specified in the conditions set
+Read more about these concepts on the [Shape and Connectivity](shape-connectivity)
+page.
 
-### Subjective Scoping
+</p>
+</li>
 
-There are currently five known sub-dimensions of subjective scoping:
+<li>
 
-1. **Travel mode**. The methods or vehicles allowed or not allowed to use a road or path.
-2. **Usage purpose**. The reason a person or entity is traveling on the transportation network. A common example of a usage purpose condition is delivery only or customer only.
-3. **Membership** in a recognized group, a.k.a. status.  Membership in a recognized group refers to the status of the person or entity traveling as recognized by authorities controlling the particular location. This includes private property or permitted portions of the transportation network.
-4. **Cargo**. Cargo refers to something the person or entity traveling on the transportation network is carrying. The most common example of a cargo scope is portions of the transportation network where transporting hazardous material is not allowed.
-5. **Vehicle Attributes**. These refer to specific attributes of vehicles that are too specific to infer from travel mode, or where it is overly cumbersome to generate a travel mode for each attribute combination. This includes scoping vehicles that are allowed based on attributes like height, weight, or axle count.
+<p>
 
-[View the Schema for a Connector Type](/reference/transportation/connector)
+**Scoped and Rule-Based Properties.** The Transportation theme schema allows property values
+to be specified for granular scopes at the sub-feature level. For example:
 
-[View the Schema for a Segment Type](/reference/transportation/segment)
+</p>
+
+- A speed limit on a road segment might be scoped to apply only to part of the
+  road geometry using Geometric Scoping.
+- The directionality of a segment, controlling the direction or directions in
```

Comment:
There is no directionality property on segment level in this schema release (not in version that this documentation tries to describe)

Reply:
Thanks. I'm taking the following actions to align the docs with the current `v.0.4.0` schema release while also being as close to our decision in https://github.com/OvertureMaps/schema-wg/issues/134 as possible:

- Add short *Heading* section and move the definition of `forward` and `backward` into this section.
- Chop down the segment directionality section to a tiny "under construction" paragraph that just says "we're working on something here".
- Move the enumeration table into the lane directionality section, which we expect to be a bit more stable.